lib/devise_saml_authenticatable.rb

```diff
@@ -19,6 +19,10 @@ class Engine < Rails::Engine
 
 # Get saml information from config/saml.yml now
 module Devise
+  # Allow route customization to avoid collision
+  mattr_accessor :route_helper_prefix
```

Comment:
Could you add a `saml_` prefix to this to avoid collisions with other plugins?

Reply:
I added the `saml_` prefix in the latest commit fd12c0b